docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml

```diff
@@ -1662,8 +1662,15 @@ dfsSSLPrivateKeyFile=/keyfilepath/keyfile</programlisting>Set the <emphasis
 
         <para>The downside of running multiple Thors on the same hardware is
         that the physical memory on the nodes needs to be shared among each of
-        the Thors. This needs to be configured per Thor cluster
-        definition.</para>
+        the Thors. This needs to be configured per Thor cluster definition.
+        </para>
+
+        <para>Multiple Thor's on the same cluster require them to share the
+        same build and installation. The environment defines each Thor
+        cluster, which can share the same machine set. There are slave and
+        master port settings that need to be set to avoid clashing. There are
+        also memory sharing/splitting considerations and settings that need to
+        be made. </para>
```

Comment:
You say there are settings to be set to avoid clashing but don't say which ones. You also say there are memory considerations but don't say what those considerations are.